packages/node-type-resolver/src/DependencyInjection/PHPStanServicesFactory.php

```diff
@@ -37,7 +37,21 @@ public function __construct(ParameterProvider $parameterProvider)
 
         $existingAdditionalConfigFiles = array_filter($additionalConfigFiles, 'file_exists');
 
-        $this->container = $containerFactory->create(sys_get_temp_dir(), $existingAdditionalConfigFiles, []);
+        // this allows to statically autoload files from Rector tests
+        $analysedPathsFromConfig = [];
+        if (defined('RECTOR_REPOSITORY')) {
+            $analysedPathsFromConfig[] = sys_get_temp_dir() . '/_temp_fixture_easy_testing';
+            // just for loading Source directory
+            $analysedPathsFromConfig[] = __DIR__ . '/../../../../rules/renaming/tests/Rector/Name/RenameClassRector/Source';
+        }
+
+        $this->container = $containerFactory->create(
+            sys_get_temp_dir(),
+            $existingAdditionalConfigFiles,
+            [],
+            [],
+            $analysedPathsFromConfig
```

Comment:
@ondrejmirtes So it seems this parameter does the trick. I passed tests fixture directory here and it works :+1: 

Now I have to replaced all `class_exists(...)` with `RelfectionProvider->hasClass(...)` :laughing: